python/cuda_parallel/CMakeLists.txt

```diff
@@ -22,3 +24,69 @@ install(
     TARGETS cccl.c.parallel
     DESTINATION cuda/parallel/experimental/cccl
 )
+
+find_package(Python3 REQUIRED COMPONENTS Development.Module)
+
+get_filename_component(_python_path ${Python3_EXECUTABLE} PATH)
+find_program(
+    CYTHON_EXECUTABLE
+    NAMES cython cython.bat cython3
+    HINTS "${_python_path}"
+    DOC "path to the cython executable"
+)
+
+set(CYTHON_version_command "${CYTHON_EXECUTABLE}" --version)
+execute_process(COMMAND ${CYTHON_version_command}
+    OUTPUT_VARIABLE CYTHON_version_output
+    ERROR_VARIABLE CYTHON_version_error
+    RESULT_VARIABLE CYTHON_version_result
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+    ERROR_STRIP_TRAILING_WHITESPACE
+)
+
+if(NOT ${CYTHON_version_result} EQUAL 0)
+    set(_error_msg "Command \"${CYTHON_version_command}\" failed with")
+    set(_error_msg "${_error_msg} output:\n${CYTHON_version_error}")
+    message(FATAL_ERROR "${_error_msg}")
+else()
+    if("${CYTHON_version_output}" MATCHES "^[Cc]ython version ([^,]+)")
+        set(CYTHON_VERSION "${CMAKE_MATCH_1}")
+    else()
+        if("${CYTHON_version_error}" MATCHES "^[Cc]ython version ([^,]+)")
+            set(CYTHON_VERSION "${CMAKE_MATCH_1}")
+        endif()
+    endif()
+endif()
+
+include(FindPackageHandleStandardArgs)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(Cython REQUIRED_VARS CYTHON_EXECUTABLE CYTHON_VERSION)
```

Comment:
It's very strange to have this outside of a find module. Probably not worth keeping, this will be checking various variables defined by `find_package` that won't be set in this scope.

Reply:
Removed, and added quotes as requested.